src/app/statistics-page/components/daily-statistics/daily-statistics.component.ts

```diff
@@ -11,19 +11,25 @@ import { secondsToMsFormat } from '../../utils/time-utils';
 })
 export class DailyStatisticsComponent implements OnInit {
   @ViewChild('picker') datePicker?: MatDatepicker<Date>;
+  @Output() selectedDayEvent = new EventEmitter<Date>();
```

Comment:
Why we should use this approach?
We send data via parent component, maybe better use shared TaskService for this.

Reply:
So we need to make app-task-list a part of dayli-statistics.component.html and change statistics-page markup?

Reply:
I don't have any explicit opinion about markup or component separation) But custom event in DailyStatistic passed data for Statistics Page, statistic page retrieve data and pass it to TaskList. In current markup its working, but we can't move DailyStatistic and TaskList to another parent component without event listening in new parent component. TaskList in current context depend from DailyStatistic, but via StatisticPage. Is better doing this from TaskService, or union TaskList with DailyStatistic in single component? I don't know